scripts/src/main/resources/scripts/command/vscode

```diff
@@ -122,10 +122,55 @@ function doRunVsCode() {
 
 function doStartVsCode() {
   doConfigureVsCode -u
+  doAddRecmdt
   doEcho "launching VSCode..."
   doRunVsCode "${WORKSPACE_PATH}"
 }
 
+function doAddRecmdt() {
+  local recmdtJSON="${WORKSPACE_PATH}/.vscode/extensions.json"
+  local file
+  local recmdtKey
+  local recmdtValue
+  local pluginsToRecmdt=()
+  for file in "${SETTINGS_PATH}"/vscode/plugins/*.properties
+  do
```

Comment:
This code block could be moved in to a function, as it is already used in the `doAddPlugins` function

Reply:
Sounds good. Maybe you can even directly integrate into `addPlugin` with a specific parameter...
For maintenance it is IMHO not so good to spare some single characters over readability (`doAddRecmdt` vs. `doAddPluginRecommendation`).
Bash and perl are designed with this philosophy of very short and compact syntax and the readability of such code is pain as you might have experienced ;)
After all our function names never need to be typed by end-users but only by our internal developers and we can use IDEs with auto-completion :)

Reply:
all done. Thanks.